### PR TITLE
use a lower case `creator` for aws tag

### PR DESCRIPTION
**Pull request by mohit**

### CODE REVIEW

**Comment by mohit** [comment]
tests were also failing on master

File: ebs_snapshots/snapshot_manager.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def _create_snapshot(connection, volume, name=''):
     if not name:
         name = '{}-snapshot'.format(volume.id)
     connection.create_tags(
-        [snapshot.id], dict(Name=name, Creator='ebs-snapshots'))
+        [snapshot.id], dict(Name=name, creator='ebs-snapshots'))
```
Comment (nathanleiby) [inline ebs_snapshots/snapshot_manager.py]: Are we keeping `Name` uppercase?

Comment (mohit) [inline ebs_snapshots/snapshot_manager.py]: `Name` is something that is used by the AWS UI and tools and it's better to just keep it that way. I just treat it as not a tag, but a property we _have_ to set.

Comment (mohit) [inline ebs_snapshots/snapshot_manager.py]: Though maybe I should add things like `application`, `env` and `version` to this. maybe that is a separate PR?

Comment (nathanleiby) [inline ebs_snapshots/snapshot_manager.py]: This one or separate is fine

**Comment by nathanleiby** [comment]
LGTM

**Comment by mohit** [comment]
merging. a more thorough tagging change, coming soon to a PR near you :)

**Pull request merged**